build/yamls/antrea-ipsec.yml

```diff
@@ -5835,6 +5982,11 @@ data:
       maxAge: 28
       # Compress enables gzip compression on rotated files.
       compress: true
+
+    bgpPolicy:
+      # The name of the Secret storing passwords of the BGP peers. For each BGP peer, the Secret key is generated by
+      # concatenating its IP address and AS number, e.g., `192.168.1.1-65521`.
+      secretName: "antrea-bgp-passwords"
```

Comment:
this is not technically part of the API, and I think we should move this, along with RBAC changes for the Secret and the Secret definition itself, to the "implementation" PR

Reply:
Make sense to me, I will move it to another PR.